uchroma/server/data/laptop.yaml

```diff
@@ -302,3 +302,97 @@ children:
     - [[5, 13], [5, 12]]  # arrow which stays where it is because
     - [[5, 14], [5, 13]]  # the up & down keys are the same cell.
 
+- !device-config
+  name: Blade (2017)
+  product_id: 0x0224
+  dimensions: [6, 16]
+  supported_fx: [disable, wave, reactive, breathe, spectrum, static, starlight,
+    rainbow, ripple_solid]
```

Comment:
Does custom_frame work? It's required to enable the animation system. I'm assuming it does if you used the key mapping tool. Rainbow effect uses custom frame as well. If it indeed works, it should be enabled.